src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceModelHttpMessageHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net;
+using System.Net.Http;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ServiceModel.Channels
+{
+    public partial class ServiceModelHttpMessageHandler : DelegatingHandler
+    {
+        public DecompressionMethods AutomaticDecompression
+        {
+            get { return _innerHandler.AutomaticDecompression; }
+            set { _innerHandler.AutomaticDecompression = value; }
+        }
+
+        public CookieContainer CookieContainer
+        {
+            get { return _innerHandler.CookieContainer; }
+            set { _innerHandler.CookieContainer = value; }
+        }
```

Comment:
How are we (or are we?) going to validate that CookieContainer is specified? Or will that be taken care of when the HttpClientHandler or WinHttpHandler is used?